azure-pipelines.yml

```diff
@@ -44,6 +44,24 @@ variables:
   value: y
 
 stages:
+- stage: Analysis
```

Comment:
Could you mark this stage not as a hard enforcement?

Reply:
I added continueOnError flag, which will continue to run even if Semgrep check is failed